microsoft/botframework-solutions
# [Enterprise Bot Template][TS] Use i18n to manage resources localization

## Description
- Implement [i18n](https://github.com/mashpie/i18n-node).
- Modify all dialogs responses implementation to use i18n.
- Delete all `.resx` files and all related references (like resx dependency).
- Add `/locales` folder with new `.json` localized files.

## Related Issue




Fixes #540 - *[Enterprise Bot Template][TS]* Use i18n to manage resources localization

## Testing Steps


1. Open a new terminal in AI/templates/Enterprise-Template/src/typescript.
1. Run `msbuild` using the instruction in [DEPLOYMENT.md](https://github.com/Microsoft/AI/blob/master/templates/Enterprise-Template/src/typescript/deploymentScripts/DEPLOYMENT.md).
1. Run `npm install` to install dependencies.
1. Run `npm run start` to run the bot.
1. Open `Bot Framework Emulator` and load the corresponding `.bot` file.
1. Change the Emulator locale to `es`.
1. Send a message to the bot. The response should be in Spanish.
1. Change the Emulator locale to `en`.
1. Send a message to the bot. The response should be in English.

## Checklist

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant

If you have updated responses or `.lu` files:
